_models/comments.models.js

```diff
@@ -0,0 +1,69 @@
+const { ident } = require("pg-format");
+const db = require("../db/connection");
+const { selectArticleById } = require("./articles.models");
+const { createRef } = require("../db/seeds/utils");
+
+exports.selectCommentsByArticleId = async (article_id) => {
+  const article = await selectArticleById(article_id);
+  if (article.error) {
+    return article;
+  } else {
+    return db
+      .query(
+        `SELECT comments.* FROM comments
+      LEFT JOIN articles ON articles.article_id = comments.article_id WHERE comments.article_id = $1 ORDER BY created_at DESC;`,
+        [article_id]
+      )
+      .then((response) => {
+        if (response.rows.length === 0) {
+          return Promise.reject({
+            error: true,
+            status: 404,
+            msg: `No comments found.`,
+          });
+        } else {
+          return response.rows;
+        }
+      });
+  }
+};
+
+/* formatComment = (username) => {
+  console.log(username);
+  return db
+    .query("SELECT username, name FROM users WHERE username = $1;", [username])
+    .then((response) => {
+      return createRef(response.rows, "username", "name");
+    });
+};
+//const userFormat = await formatComment(comment.username);
+//comment["username"] = userFormat[comment.username]; */
```

Comment:
Originally thought I needed to use the createRef func to insert the name of the user not the username - keeping the code commented out again incase it comes of use.

Reply:
Pop it in another file that is git ignored. Main branch should be sacrosanct. In continuous development your remote main might literally be to code that gets pushed out to the client's devices